src/main/java/gtPlusPlus/xmod/gregtech/common/tileentities/misc/TileEntitySolarHeater.java

```diff
@@ -225,8 +225,8 @@ public boolean isEnetOutput() {
 
 	@Override
 	public void saveNBTData(NBTTagCompound aNBT) {
```

Comment:
You can remove NBT handling if `mHasTower` is always written as false

Reply:
I'll comment out the NBT handling, instead. It might be useful once this code is refactored, but right now the reflector NBT prevented the multi from working correctly on leaving and entering world.